### PR TITLE
Feature(cli) GRID-8908: Added docs about `--no-copy` for CLI datastore creation flow

### PR DESCRIPTION
**Pull request by rlizzo**
# What does this PR do?

Added docs about `grid datastore create s3://foo --no-copy` option when creating datastores from public s3 buckets via the CLI.

### CODE REVIEW

File: docs/features/datastores/2_Using Datastores/2_creating-datastores.md [hunk 1]
```diff
@@ -102,6 +102,27 @@ grid datastore create s3://ryft-public-sample-data/esRedditJson/ --name lightnin
...
+
+When using this flag, you cannot remove files from your bucket. If you'd like to add.
+files, please create a new version of the datastore after you've added files to your.
+bucket. Please note that Grid does not currently support private S3 buckets. 
+
+If using this flag via the grid public cloud, then the source bucket should be in the 
+AWS `us-east-1` region or there will be significant latency when you attempt to access
+the datastore files in a run / session. 
+
+:::
```
Comment (essiequoi) [inline docs/features/datastores/2_Using Datastores/2_creating-datastores.md:124]:
```suggestion
In certain cases, your s3 bucket may fit one (or both) of the following criteria: (1)
the bucket is continually updating with new data which you want included in a Grid datastore (2) the bucket is
particularly large (leading to long Datastore creation times). In these cases,
you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will
prevent grid from making a copy of the dataset, speeding up datastore creation time
significantly when working with large buckets. 

:::info 

When using this flag, you cannot remove files from your bucket. If you'd like to add files, please create a new version of the datastore after you've added files to your bucket. Please note that Grid does not currently support private S3 buckets. 

If using this flag via the grid public cloud, then the source bucket should be in the  AWS `us-east-1` region or there will be significant latency when you attempt to access the datastore files in a run / session. 

:::
```

**Review by essiequoi — APPROVED** [review]

**Review by essiequoi — APPROVED** [review]

**Comment by rlizzo** [comment]
@robert-s-lee can you review?

File: docs/features/datastores/2_Using Datastores/2_creating-datastores.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ For datasets larger than 1 GB, use the CLI.
 
 :::note 
 
-If you have a dataset that is 1Gb+ and poor-average internet connection, we suggest starting an Interactive Session and creating the Datastore from there. Internet speed is much faster in Interactive Sessions, so upload times will be shorter. 
+If you have a dataset that is 1GB+ and poor-average internet connection, we suggest starting an Interactive Session and creating the Datastore from there. Internet speed is much faster in Interactive Sessions, so upload times will be shorter.
```
Comment (RobertLaurella) [inline docs/features/datastores/2_Using Datastores/2_creating-datastores.md:34]:
```suggestion
We suggest starting an Interactive Session and creating your Datastore from there in the following instances:

- Your dataset is larger than 1GB
- You do not have a strong internet connection

Using an Interactive Session in this scenario will improve your upload speed.

```

File: docs/features/datastores/2_Using Datastores/2_creating-datastores.md [hunk 3]
```diff
@@ -102,6 +102,26 @@ grid datastore create s3://ryft-public-sample-data/esRedditJson/ --name lightnin
...
+particularly large (leading to long Datastore creation times). In these cases,
+you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will
+prevent grid from making a copy of the dataset, speeding up datastore creation time
+significantly when working with large buckets. 
+
+:::info 
+
+When using this flag, you cannot remove files from your bucket. If you'd like to add files, please create a new version of the datastore after you've added files to your bucket. Please note that Grid does not currently support private S3 buckets. 
+
+If using this flag via the grid public cloud, then the source bucket should be in the  AWS `us-east-1` region or there will be significant latency when you attempt to access the datastore files in a run / session.
```
Comment (RobertLaurella) [inline docs/features/datastores/2_Using Datastores/2_creating-datastores.md:121]:
```suggestion
Example: 
`grid datastore create S3://ruff-public-sample-data/esRedditJson --no-copy`

In certain cases, your S3 bucket may fit one (or both) of the following criteria:

(1) the bucket is continually updating with new data which you want included in a Grid Datastore 
(2) the bucket is particularly large (leading to long Datastore creation times)

In these cases, you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will
prevent Grid from making a copy of the dataset, which significantly speeds up Datastore creation time.

:::info 

When using this flag, you cannot remove files from your bucket. If you'd like to add files, please create a new version of the Datastore after you've added files to your bucket. 

Please note that Grid does not currently support private S3 buckets. 

If you are using this flag via the Grid public cloud, then the source bucket should be in the  AWS `us-east-1` region or there will be significant latency when you attempt to access the Datastore files in a Run or Session.
```

Comment (RobertLaurella) [reply docs/features/datastores/2_Using Datastores/2_creating-datastores.md:121]: NB: I am capitalizing all of the following words: Grid, Run, Session, Datastore - to my knowledge these are Grid-specific words (as opposed to eg 'dataset') and should be capitalized. Happy to iterate further, but this my logic behind what gets capitalized in the docs etc

**Review by RobertLaurella — CHANGES_REQUESTED** [review]

**Review by RobertLaurella — APPROVED** [review]

**Pull request merged**